app/ssttpaffordability/AffordabilityController.scala

```diff
@@ -210,4 +210,9 @@ class AffordabilityController @Inject() (
     }
   }
 
+  def getWeCannotAgreeYourPP: Action[AnyContent] = as.action { implicit request =>
+    JourneyLogger.info(s"$request")
```

Comment:
`s"$request"` is the same as `request` :-) , 
You may want to prefix it with something like "getWeCannotAgreeYourPP: $request"

Reply:
👍🏻